web/docs/guides/with-sveltekit.mdx

```diff
@@ -390,8 +390,6 @@ npm run dev
 
 And then open the browser to [localhost:3000](http://localhost:3000) and you should see the completed app.
 
-> ⚠️ WARNING: Svelte uses by default `port 5000`, Supabase uses `port 3000`. To change the redirection port for supabase go to: `Authentication > Settings` and change the `Site Url` to `localhost:5000`
```

Comment:
Is there a reason for removing the this warning?

Reply:
As mentioned above, both Supabase and Svelte Kit use port 3000 as the default